.vib/rabbitmq/cypress/cypress/integration/rabbitmq_spec.js

```diff
@@ -0,0 +1,102 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows login/logout', () => {
+  const APP_NAME = 'RabbitMQ';
+
+  cy.login();
+  cy.get('#versions').should('contain', APP_NAME);
+  cy.contains('.hider', 'Queued messages');
+  cy.contains('Log out').click();
+  cy.contains('#login', 'Username');
+});
+
+it('allows upload of broker definitions', () => {
+  cy.login();
+  cy.contains('Import definitions').click();
+  cy.get('input[type="file"]').selectFile(
+    'cypress/fixtures/broker-definitions.json'
+  );
+  cy.contains('Upload broker definitions').click();
+  cy.get('.form-popup-warn').should('not.exist');
+  cy.contains('Your definitions were imported successfully');
+});
+
+it('allows publishing a message to a created exchange', () => {
+  cy.login();
+  cy.visit('#/exchanges');
+  cy.contains('Add a new exchange').click();
+  cy.fixture('exchanges').then((exchange) => {
+    cy.get('[name="name"]')
+      .scrollIntoView()
+      .type(`${exchange.newExchange.name}${random}`);
+    cy.get('#arguments_1_mfkey').type(exchange.newExchange.argument1);
+    cy.get('#arguments_1_mfvalue').type(exchange.newExchange.argument2);
+    cy.contains('Add exchange').click({ force: true });
+    cy.get('table[class="list"]').should(
+      'contain',
+      `${exchange.newExchange.name}${random}`
+    );
+    cy.contains('a', `${exchange.newExchange.name}${random}`).click();
+  });
+  cy.contains('Publish message').click();
+  cy.fixture('messages').then((message) => {
+    cy.get('textarea').type(message.newMessage.payload);
+  });
+  cy.contains('input', 'Publish message').click();
+  cy.contains('Message published, but not routed');
+});
+
+it('allows adding a new queue and binding/unbinding it to the exchange', () => {
+  cy.login();
+  cy.visit('#/queues');
+  cy.contains('Add a new queue').click();
+  cy.fixture('queues').then((queue) => {
+    cy.get('[name="name"]').type(`${queue.newQueue.name}${random}`);
+    cy.get('#arguments_1_mfkey').type(queue.newQueue.argument1);
+    cy.get('#arguments_1_mfvalue').type(queue.newQueue.argument2);
+    cy.contains('Add queue').click();
+    cy.get('table[class="list"]').should(
+      'contain',
+      `${queue.newQueue.name}${random}`
+    );
```

Comment:
Could we use `cy.contains` here?
```suggestion
    cy.contains('table', `${queue.newQueue.name}${random}`);
    ---
    cy.contains('[class="list"]', `${queue.newQueue.name}${random}`);
```

This will apply to [L90](https://github.com/bitnami/charts/pull/10102/files#diff-11ca9508e8abe677c1ee5da0453302f35fd31855171af5ba810dd34865b89934R90) as well.